### PR TITLE
feat: storybook for lib

### PR DESCRIPTION
**Pull request by shivadharmi**
This pull request includes a configuration of the storybook for lib

Closes https://github.com/helpwave/web/issues/375

### CODE REVIEW

**Comment by MaxSchaefer** [comment]
@shivadharmi thank you very much! I'm missing our fonts defined at https://github.com/helpwave/web/blob/main/tasks/pages/_app.tsx#L12. Is it possible to add them to Storybook as well?

File: README.md [hunk 1]
```diff
@@ -2,20 +2,26 @@
...
 1. `pnpm run dev`
 1. open [`http://localhost:3000`](http://localhost:3000)
 
-# Testing
+## Storybook
+
+We are about to adopting Storybook to develop, preview, testing and showcasing our components.
+
+- [`lib` https://storybook-lib.helpwave.de](https://storybook-lib.helpwave.de)
+
```
Comment (DasProffi) [inline README.md:21]: Can we add a way to show the storybook locally or add the url to do that here?

Comment (MaxSchaefer) [reply README.md]: @DasProffi of course. The ability is already there, I've improved the README.

**Review by DasProffi — APPROVED** [review]

**Comment by shivadharmi** [comment]
> @shivadharmi thank you very much! I'm missing our fonts defined at https://github.com/helpwave/web/blob/main/tasks/pages/_app.tsx#L12. Is it possible to add them to Storybook as well?

Hey @MaxSchaefer I didn't see this comment. I didn't receive the notification 😕. So sorry.

**Comment by MaxSchaefer** [comment]
> > @shivadharmi thank you very much! I'm missing our fonts defined at https://github.com/helpwave/web/blob/main/tasks/pages/_app.tsx#L12. Is it possible to add them to Storybook as well?
> 
> Hey @MaxSchaefer I didn't see this comment. I didn't receive the notification 😕. So sorry.

No problem. We talked about moving this pr to me. Feel free to review 😀

**Pull request merged**